packages/ember-runtime/lib/system/array_proxy.js

```diff
@@ -77,7 +74,23 @@ var ArrayProxy = EmberObject.extend(MutableArray, {
     @type Ember.Array
     @private
   */
-  content: null,
+  content: computed({
+    get() {
+      return this._content;
+    },
+    set(k, v) {
+      if (this._didInitArrayProxy) {
+        var oldContent = this._content;
+        var len = oldContent ? get(oldContent, 'length') : 0;
+        this.arrangedContentArrayWillChange(this, 0, len, undefined);
+        this.arrangedContentWillChange(this);
+      }
+      this._content = v;
+      return v;
+    }
+  }),
+
```

Comment:
People who were setting `content` in `extend` will have a bad time. But I would argue that is nuts anyway, because it causes all your instances to share the same array.